public/js/test/mochitest/head.js

```diff
@@ -204,3 +221,27 @@ function isVisibleWithin(outerEl, innerEl) {
   return innerRect.top > outerRect.top &&
     innerRect.bottom < outerRect.bottom;
 }
+
+
+const selectors = {
+  callStackArrow: ".call-stack-pane .arrow",
+  frame: (index) => `.frames ul li:nth-child(${index})`
+}
```

Comment:
This is awesome. I was thinking about something like this as well instead of having a ton of `get*` functions.